frontend/src/components/sideBar/sideBarData.jsx

```diff
@@ -0,0 +1,39 @@
+import * as CiIcons from "react-icons/ci";
```

Comment:
Same import multiple times here

Reply:
All paths are different

Reply:
All paths are differents, we tried to import them in a single line but it didn't work.